src/components/Search/Search.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../styles/Mixins.scss';
+@import '../../styles/Breakpoints.scss';
+@import '../../styles/Colors.scss';
+
+.search {
+  display: flex;
```

Comment:
maybe you could use flexCenter

Reply:
Now i remember i tried flexCenter but it looked like this 
![image](https://user-images.githubusercontent.com/87152087/126641871-e83193e4-756f-4efe-9c14-bcb7ec60d94b.png)
it looked fine in the header, but not as a separate component

Reply:
it's ok for me :)